sys/sys/msg.h

```diff
@@ -146,6 +146,8 @@ struct msqid_kernel_kvm {
 	struct	msqid_ds u;
 	struct	label *label;	/* MAC label */
 	struct	ucred *cred;	/* creator's credentials */
+	struct	msg *first_msg;
+	struct	msg *last_msg;
```

Comment:
Maybe make this a STAILQ_HEAD() while you are here?  Would have to turn msg_next into a STAILQ_LINK.

Reply:
I had the same though after I submitted the PR on Friday.